MangopayVaultSDK/MangopayVault.swift

```diff
@@ -85,7 +96,11 @@ public class MangopayVault {
                 _card.accessKeyRef = _cardRegistration.accessKey
                 _card.data = _cardRegistration.preregistrationData
 
-                let redData = try await self.paylineClient!.postCardInfo(_card, url: url)
+                let redData = try await self.paylineClient!.postCardInfo(
```

Comment:
@elviva-mgp  let's rename payLineClient to something generic (e.g vaultClient) as we now support different providers